sentences/de/cover_HassTurnOn.yaml

```diff
@@ -17,22 +17,26 @@ intents:
           - <öffnen> <area> (<abdeckung>|<tor>|<garage>)
           - <öffnen> (<abdeckung>|<tor>|<garage>) <area>
           - (<abdeckung>|<tor>|<garage>) <area> <öffnen>
-          - <machen> <area> (<abdeckung>|<tor>|<garage>) <auf>
-          - <machen> (<abdeckung>|<tor>|<garage>) <area> <auf>
-          - (<abdeckung>|<tor>|<garage>) <area> <auf>[ <machen>]
+          - (<machen>|<fahren>) <area> (<abdeckung>|<tor>|<garage>) <auf>
+          - (<machen>|<fahren>) (<abdeckung>|<tor>|<garage>) <area> <auf>
+          - (<abdeckung>|<tor>|<garage>) <area> <auf>[ (<machen>|<fahren>)]
         slots:
           domain: cover
         response: cover
 
       - sentences:
           - <öffnen> <area> (<alle_tore>|<alle_garagen>)
           - <öffnen> (<alle_tore>|<alle_garagen>) <area>
-          - <machen> <area> (<alle_tore>|<alle_garagen>) <auf>
-          - <machen> (<alle_tore>|<alle_garagen>) <area> <auf>
+          - (<machen>|<fahren>) <area> (<alle_tore>|<alle_garagen>) <auf>
+          - (<machen>|<fahren>) (<alle_tore>|<alle_garagen>) <area> <auf>
           - <öffnen>[ <area>] (<alle_abdeckungen>|<alle_tore>|<alle_garagen>)
           - <öffnen> (<alle_abdeckungen>|<alle_tore>|<alle_garagen>)[ <area>]
-          - <machen>[ <area>] (<alle_abdeckungen>|<alle_tore>|<alle_garagen>) <auf>
-          - <machen> (<alle_abdeckungen>|<alle_tore>|<alle_garagen>)[ <area>] <auf>
+          - (<machen>|<fahren>)[ <area>] (<alle_abdeckungen>|<alle_tore>|<alle_garagen>) <auf>
+          - (<machen>|<fahren>) (<alle_abdeckungen>|<alle_tore>|<alle_garagen>)[ <area>] <auf>
+          - <fahren> <area> (<alle_tore>|<alle_garagen>) <auf>
+          - <fahren> (<alle_tore>|<alle_garagen>) <area> <auf>
+          - <fahren>[ <area>] (<alle_abdeckungen>|<alle_tore>|<alle_garagen>) <auf>
+          - <fahren> (<alle_abdeckungen>|<alle_tore>|<alle_garagen>)[ <area>] <auf>
```

Comment:
```suggestion
```
these new sentences are already covered by the both sentences above

Reply:
Thanks, leftover from when I did not yet combine <machen> with <fahren>.